recipe/meta.yaml

```
@@ -34,37 +47,21 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - arrow-cpp {{ version }}
-    - cython >=0.29.31
+    - cython >=3
     - numpy 2.0 # [py<313]
     - numpy 2.1 # [py>=313]
     - pip
     - python
-    - setuptools
-    - setuptools_scm
+    - setuptools >=64
+    - setuptools_scm >=8
```

https://github.com/apache/arrow/blob/apache-arrow-21.0.0/python/pyproject.toml#L29
```suggestion
    - setuptools_scm >=8
    - toml
```